frontend/packages/knative-plugin/src/actions/getEventPubSubActions.ts

```diff
@@ -0,0 +1,17 @@
+import { Kebab } from '@console/internal/components/utils';
+import { ModifyApplication } from '@console/dev-console/src/actions/modify-application';
+import { K8sKind } from '@console/internal/module/k8s';
+import { getKebabActionsForKind } from '../utils/kebab-actions';
+import { EventingBrokerModel } from '../models';
+
+export const getEventPubSubActions = (model: K8sKind) => {
+  let commonActions = Kebab.factory.common;
+
+  if (model.kind === EventingBrokerModel.kind) {
+    commonActions = Kebab.factory.common.filter((action) => {
+      return !['Delete', 'Edit'].includes(action.name);
```

Comment:
@karthikjeeyar will this filter out `delete` and `edit` for broker?

Reply:
Yes, this is to filter those two common actions

Reply:
we don't need to filter karthik , we decided to have these as common actions based on user permissions i.e edit label/annotations are essentially edit broker